src/Symfony/Component/Translation/Bridge/Lokalise/LokaliseProviderFactory.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Bridge\Lokalise;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Translation\Exception\UnsupportedSchemeException;
+use Symfony\Component\Translation\Loader\LoaderInterface;
+use Symfony\Component\Translation\Provider\AbstractProviderFactory;
+use Symfony\Component\Translation\Provider\Dsn;
+use Symfony\Component\Translation\Provider\ProviderInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Mathieu Santostefano <msantostefano@protonmail.com>
+ *
+ * @experimental in 5.3
+ */
+final class LokaliseProviderFactory extends AbstractProviderFactory
+{
+    private const HOST = 'api.lokalise.com';
+
+    private $client;
+    private $logger;
+    private $defaultLocale;
+    private $loader;
+
+    public function __construct(HttpClientInterface $client, LoggerInterface $logger, string $defaultLocale, LoaderInterface $loader)
+    {
+        $this->client = $client;
+        $this->logger = $logger;
+        $this->defaultLocale = $defaultLocale;
+        $this->loader = $loader;
+    }
+
+    /**
+     * @return LokaliseProvider
+     */
+    public function create(Dsn $dsn): ProviderInterface
+    {
+        if ('lokalise' !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, 'lokalise', $this->getSupportedSchemes());
+        }
+
+        $endpoint = sprintf('%s%s', 'default' === $dsn->getHost() ? self::HOST : $dsn->getHost(), $dsn->getPort() ? ':'.$dsn->getPort() : '');
+        $client = $this->client->withOptions([
+            'base_uri' => sprintf('https://%sprojects/%s', $endpoint, $this->getUser($dsn)). '/api2/',
```

Comment:
```suggestion
            'base_uri' => sprintf('https://%sprojects/%s/api2/', $endpoint, $this->getUser($dsn)),
```

Reply:
My bad, I didn't understand "fixed part". It's fixed now. Sorry.